.rubocop.yml

```diff
@@ -59,6 +59,7 @@ Lint/RescueWithoutErrorClass:
 
 Metrics/AbcSize:
   Exclude:
+    - 'app/jobs/replication_check_job.rb' # [18.49/15] method still readable
```

Comment:
ah crap did the squash wrong (i'm just going to leave this here) 
......this should actually be `endpoint_check_job.rb` since the name changes in the last commit...but im not sure what i did here